### ISSUE TITLE
Cannot set some assets as fee payment asset

### ISSUE THREAD

**Issue by lolmcshizz** [galacticcouncil/Hydra-mining-ui#39]
When selecting the "3 dots" next to an asset, a small pink dot appears instead of the "set as fee payment asset" and therefore it is not possible to set e.g. DOT as the fee payment asset.

![image](https://user-images.githubusercontent.com/97540577/206180792-93494e4c-0545-4ee5-9e50-f8fae24a1cbb.png)

This appears to be an issue only with the bottom asset (DOT) - as the box appears for any other asset that is not currently the fee payment asset:

![image](https://user-images.githubusercontent.com/97540577/206180997-25308dc9-c7c8-47cf-9d52-9efe0cbfe1e7.png)

We should consider whether we need some kind of empty state for when it is not possible to set as fee payment asset instead of a tiny pink dot

**Comment by jak-pan** [comment]
This is correct. DOT is currently not setup as fee payment asset on-chain. But we should fix the state to disabled if there is no action

**Comment by tomashapl1** [comment]
@lolmcshizz Could you retest ? we disabled "dots" in case there is not action available

**Comment by lolmcshizz** [comment]
Hi @tomashapl1 can confirm the fix works!
